build.gradle

```diff
@@ -34,10 +36,17 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.json', name: 'json', version: '20180130'
+    compile group: 'org.json', name: 'json', version: '20180813'
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile group: 'it.sauronsoftware', name: 'junique', version: '1.0.4'
-    compile "de.codecentric.centerdevice:centerdevice-nsmenufx:2.1.6"
+    compile group: 'it.sauronsoftware', name: 'junique', version: '1.0.4' //for only one copy of blobsaver at a time
+    compile "de.codecentric.centerdevice:centerdevice-nsmenufx:2.1.6" //for macOS menu bar
+//    compile 'com.google.code.gson:gson:2.8.5' // for app preferences TODO: implement this
```

Comment:
Why Gson? I'm already using org.json?

Reply:
OK, I'll remove it

Reply:
I just wanted to store custom object files

Reply:
Wdym?